integration-tester/tests/100-logs/logs_test.go

```
@@ -8,8 +8,9 @@ import (
 
```

I don't think the ignoring errors change should be in its own commit since the integration tests would be broken between this commit, and the commit that adds the test. It's borderline though, if you think the single commit would be too big then I'm fine having it separate.